README.md

```diff
@@ -103,29 +104,40 @@ We use Bun secrets for local development and SST secrets for remote deployments.
 
 Secret names:
 
-- LLM:
+- Auth
+  - GitHub: `GithubClientId`, `GithubClientSecret`
+  - Google: `GoogleClientId`, `GoogleClientSecret`
+  - Microsoft: `MicrosoftClientId`, `MicrosoftClientSecret`
+  - Email OTP and Magic Link: `SmtpHost`, `SmtpPort`, `SmtpUser`, `SmtpPass`, `SmtpFrom`
+- LLMs
   - Bedrock: `BedrockRoleArn`, `BedrockRegion`
   - Vertex: `VertexServiceAccountEmail`, `VertexAwsProviderAudience`, `VertexProject`, `VertexLocation`
   - Others: `CohereApiKey`, `GroqApiKey`
-- Auth (Stack Auth): `StackSecretServerKey`, `StackPublishableClientKey`, `StackProjectId`
+
+Note for local development: if **auth is enabled** but SMTP secrets are not configured, the email OTP is logged to the console (look for `>>> OTP:`) so you can sign in without setting up an email provider.
 
 Local (Bun) examples:
 
 ```bash
 # set / get / delete
-bun run secret set StackSecretServerKey <value>
-bun run secret get StackSecretServerKey
-bun run secret delete StackSecretServerKey
+bun run secret set GithubClientId <value>
+bun run secret get GithubClientId
+bun run secret delete GithubClientId
 ```
 
 Remote (SST) examples:
 
 ```bash
 # set / remove (choose your <stage>)
-bun run sst secret set StackSecretServerKey <value> --stage <stage>
-bun run sst secret remove StackSecretServerKey --stage <stage>
+bun run sst secret set GithubClientId <value> --stage <stage>
+bun run sst secret remove GithubClientId --stage <stage>
 ```
 
+Auth middleware toggle:
```

Comment:
To follow our existing conventions in terms of 0 to 60 DX experience, shouldn't this be activated automatically for `bun run dev` on the monorepo level, but deactivated for `bun run -F @hebo/console dev`? (Same as we don't have to "enable" gateway or API)